tests/integration-tests/configs/develop.yaml

```diff
@@ -19,12 +19,6 @@ test-suites:
           oss: ["{{ os }}"]
           schedulers: ["{{ scheduler }}"]
         {%- endfor %}
-        {%- for os, scheduler in [("ubuntu1804", "sge"), ("alinux2", "slurm")] %}
```

Comment:
Is it possible to comment this out rather than removing it so it's easier to restore later?